sadu-updater/src/main/java/de/chojo/sadu/updater/SqlUpdater.java

```diff
@@ -127,10 +127,10 @@ private SqlUpdater(DataSource source, QueryBuilderConfig config, String versionT
      * @throws IOException if the version file does not exist.
      */
     @CheckReturnValue
-    public static <T extends JdbcConfig<?>> SqlUpdaterBuilder builder(DataSource dataSource, Database<T> type) throws IOException {
+    public static <T extends JdbcConfig<?>> SqlUpdaterBuilder<T> builder(DataSource dataSource, Database<T> type) throws IOException {
         var version = "";
         try (var versionFile = SqlUpdater.class.getClassLoader().getResourceAsStream("database/version")) {
-            version = new String(versionFile.readAllBytes()).trim();
+            version = new String(versionFile.readAllBytes(), StandardCharsets.UTF_8).trim();
```

Comment:
This is solely for completeness. The version file will only contain numbers and a single dot. Therefore this is not breaking. It can be considered code documentation.